nselib/data/http-fingerprints.lua

```diff
@@ -7044,7 +7044,90 @@ table.insert(fingerprints, {
 ----        Open Source CMS checks          ----
 ------------------------------------------------
 
--- Broad wordpress version identification
+-- Wordpress versions (Scraping readme file)
+table.insert(fingerprints, {
+  category = 'cms',
+  probes = {
+    {
+      path = '/readme.html',
+      method = 'GET'
+    }
+  },
+  matches = {
+    {
+      match = '[V|v]ersion ([0-9 .]*)',
+      output = 'Wordpress version: \\1'
+    }
+  }
+});
+
+-- Wordpress versions (Scraping metatags and wp-includes)
+table.insert(fingerprints, {
+  category = 'cms',
+  probes = {
+    {
+      path = '/',
+      method = 'GET'
+    }
+  },
+  matches = {
+    {
+      match = '<meta name="generator" content="WordPress ([0-9 .]*)" />',
+      output = 'WordPress version: \\1'
+    },
+    {
+      match = '/wp-includes\\/js\\/wp-emoji-release.min.js?ver=([0-9 .]*)',
```

Comment:
No, Dan. We can't use `"/wp-includes/js/[%w.-]+.js?ver=([0-9.]+)"` because there are CSS and JS files which are linked in WordPress through external scripts.

For example,
`"/wp-includes/js/jquery/jquery.js?ver=1.7.1"` also matches the regex proposed above. But this refers to Jquery version not WordPress version, so I think its better to use string instead of regex.